src/osd/PG.cc

```diff
@@ -970,7 +970,7 @@ PG::Scrubber::~Scrubber() {}
  *  3) Prefer current primary
  */
 map<pg_shard_t, pg_info_t>::const_iterator PG::find_best_info(
-  const map<pg_shard_t, pg_info_t> &infos) const
+  const map<pg_shard_t, pg_info_t> &infos, bool *history_les_bound) const
```

Comment:
Assert that the pointer is non-null at the top or check for null before assigning -- someone's going to try to call this with a nullptr on the assumption that it's optional.